config/plugins/interactive_environments/neo/config/neo.ini.sample

```diff
@@ -0,0 +1,43 @@
+[main]
+# Following options are ignored if using the Galaxy dynamic proxy but
+# are useful if mapping a range of ports for environment consumption.
+#password_auth = False
+#ssl = False
+
+[docker]
+# Command to launch docker container. For example `sudo docker` or `docker-lxc`.
+# If you need to use a command like `sg` you can do that here, just be sure to
+# wrap all of the docker portion in single quotes. E.g. `sg 'docker' 'docker {docker_args}'`
+#
+# It is recommended that you use command_inject if you need to inject
+# additional parameters. This command string is re-used for a `docker inspect`
+# command and will likely cause errors if it is extensively modified, past the
+# usual group/sudo changes.
+#command = docker {docker_args}
+
+# The image argument was moved to "allowed_images.yml.sample"
+
+# Additional arguments that are passed to the `docker run` command.
+# NEO4J_UID=1371, NEO4j_GIG=999
+command_inject = --sig-proxy=true -e DEBUG=false -e DEFAULT_CONTAINER_RUNTIME=120 -e NEO4J_UID=none -e NEO4J_GID=none
```

Comment:
Thank you, that is helpful. This way people could at least launch this without changing configuration.